Revise Swift entries (#929)
Codecademy/docs: content/swift/concepts/loops/loops.md

---
Title: 'Loops'

> Description: 'The for-in loop is used to iterate over collections, including strings and ranges.'

Reviewer: "for - in loop should be bolded below.

Author: @SSwiniarski I'm opting for a general loops description here

The output would be:

```shell
s
u
p
e
r
```

### Ranges

Ranges created by the `...` operator will include the numbers from the lower bound to (and includes) the upper bound.

```swift
let zeroToThree = 0...3

// zeroToThree: 0, 1, 2, 3
```

### Using Underscore

> Use `_` instead of a placeholder variable if the variable is not referenced in the `for`-`in` loop body.

Reviewer 2: ```suggestion
An underscore `_` is used instead of a placeholder variable if the variable is not referenced in the `for`-`in` loop body.
```

The output would be:

```shell
s
u
p
e
r
```

### Ranges

> Ranges created by the `...` operator will include the numbers from the lower bound to (and includes) the upper bound.

Reviewer: ```suggestion
Ranges created by the `...` operator will include the numbers from the lower bound to (and including) the upper bound.
```

The output would be:

```shell
Olé
Olé
Olé
```

### `stride()` Function

> Calling `stride()` with the 3 necessary arguments creates a collection of numbers; the arguments decide the starting number to, the (excluded) ending number, and how to increment/decrement from the start to the end.

Reviewer: ```suggestion
Calling `stride()` with the 3 necessary arguments creates a collection of numbers; the arguments decide the starting number to the (excluded) ending number, and how to increment/decrement from the start to the end.
```
